lnbits/core/models.py

```diff
@@ -221,7 +246,7 @@ def from_row(cls, row: Row):
             bolt11=row["bolt11"] or "",
             preimage=row["preimage"] or "0" * 64,
             extra=json.loads(row["extra"] or "{}"),
-            pending=row["pending"],
+            status=row["status"],
```

Comment:
for backwards compatibility:

```suggestion
pending=row["status"] == PaymentState.PENDING.value,
```

see: https://github.com/lnbits/lnbits/pull/2537#issuecomment-2136880675